docker-storage-setup.sh

```diff
@@ -978,7 +976,7 @@ setup_extra_volume() {
 }
 
 setup_extra_lv_fs() {
-  [ -z "$CONTAINER_ROOT_LV_MOUNT_PATH" ] && [ -z "$CONTAINER_ROOT_LV_NAME" ] && return 0
+  [ -z "$RESOLVED_MOUNT_DIR_PATH" ] && [ -z "$CONTAINER_ROOT_LV_NAME" ] && return 0
```

Comment:
I think this is confusing. Given we have already verified that both CONTAINER_ROOT_MOUNT_DIR_PATH and CONTAINER_ROOT_LV_NAME have been passed in, why not make just a single check.

`[ -z $RESOLVED_MOUNT_DIR_PATH ] && return 0`